### ISSUE TITLE
ccgan wrong input for discriminator

### ISSUE THREAD

**Issue by mrnameless123** [eriklindernoren/Keras-GAN#5]
on the subfolder of ccgan, I notice that you set valid image to both discrimination of valid images and fake images

` # Train the discriminator

      d_loss_real = self.discriminator.train_on_batch(imgs, [valid, labels], class_weight=class_weights)

      d_loss_fake = self.discriminator.train_on_batch(imgs, [valid, fake_labels], class_weight=class_weights)
      d_loss = 0.5 * np.add(d_loss_real, d_loss_fake)`
is that correct or you intended to do so?

**Comment by eriklindernoren** [comment]
Thanks!
